packages/css/label.css

```diff
@@ -8,39 +8,39 @@
     color: var(--fds-semantic-text-neutral-default);
   }
 
-  .label.spacing {
+  .fds-label.fds-label--spacing {
```

Comment:
Maybe just remove `.fds-label` from the next lines? they are kinda redundant.

Reply:
removed 👍🏻